### ISSUE TITLE
Enter key settings 

### ISSUE THREAD

**Issue by moisesbelda** [libretro/hatari#100]
When I use enter key , hatari settings window is displayed. I need to use the enter key as enter key.

**Comment by hizzlekizzle** [comment]
Is that with game focus mode enabled? If so, have you gone into quick menu > controls > port 1 controls to try and change the mapping?

**Comment by moisesbelda** [comment]
Yes that is with game focus enabled , which means that all hotkeys are disabled. They are all disabled (fast forward , virtual keyboard) except enter which displays me the hatari specific settings screen where you can specify ram, processor etc of the Atari st.

In quick settings - control - port  1, I have mapped the retropad with the common keys (up, down, left etc) but there is no reference to Enter.’ Key or hatari emulator settings screen. I have checked port 2 and the same 

is it the normal behavior?

**Comment by greenchili2** [comment]
Sounds like you are using an older version of the core.  Have you tried updating the core?

Also what platform are you running this on.  Retroarch is available for many platforms and it helps to know which you are running on.

**Comment by benoitrolland** [comment]
Same problem on recalbox 9.1-Pulstar x86_64
